core/src/components/select/test/label/select.e2e.ts

```diff
@@ -171,7 +171,7 @@ configs().forEach(({ title, screenshot, config }) => {
       test('label should appear on top of the select when the select is expanded', async ({ page }) => {
         await page.setContent(
           `
-           <ion-select class="select-expanded" label="Label" label-placement="floating" placeholder="Select a Fruit">
+           <ion-select class="select-expanded label-floating" label="Label" label-placement="floating" placeholder="Select a Fruit">
```

Comment:
Since I moved all the floating styles to the `label-floating` class, the `select-expanded` class by itself is no longer sufficient to show the behavior. The change makes the test reflect the actual behavior (both classes are added).

Reply:
Make sense, thanks!